vllm/v1/engine/core.py

```diff
@@ -344,6 +345,8 @@ def signal_handler(signum, frame):
             if engine_core is not None:
                 engine_core.shutdown()
 
+    @profile(save_file=envs.VLLM_PROFILE_ENGINE_CORE_LOOP,
+             enabled=envs.VLLM_PROFILE_ENGINE_CORE_LOOP != "0")
```

Comment:
I'm not really sure if it makes sense to include the env var or not. It's almost as easy to just add this line as needed, and there's lots of ways and tools to use ... not just this one

I think I've convinced myself to drop this from the PR